### PR TITLE
Allow resizing UVEditor action window from any corner,sides.

### PR DESCRIPTION
**Pull request by MartinImgSpc**

### CODE REVIEW

File: Editor/EditorCore/EditorGUILayout.cs [hunk 1]
```diff
@@ -120,32 +121,118 @@ class ResizeHandleState
             public Rect startingRect;
         }
 
-        public static Rect DoResizeHandle(Rect rect)
+        private delegate Rect ComputeResize(Rect curretRect, ResizeHandleState handleState, Event currentEvent);
```
Comment (karl-) [inline Editor/EditorCore/EditorGUILayout.cs]: style guide - omit implicit access modifiers

Comment (karl-) [reply Editor/EditorCore/EditorGUILayout.cs]: `Rect curretRect` <- `currentRect`

File: Editor/EditorCore/EditorGUILayout.cs [hunk 2]
```diff
@@ -120,32 +121,118 @@ class ResizeHandleState
...
+            return currentRect;
+        }
+
+        static int s_ResizeHandleAreaDimension = 6;
+        static int s_MoveWindowAreaHeight = 30;
+
+        public static Rect DoResizeHandle(Rect rect)
+        {
+            var evt = Event.current;
+            if (evt.rawType == UnityEngine.EventType.Used)
```
Comment (karl-) [inline Editor/EditorCore/EditorGUILayout.cs]: Why use `rawType` here? I would expect you'd want the `type`.

**Comment by MartinImgSpc** [comment]
@karl- and other reviewers. I will do the change proposed. However when we have done supplemental testing we discovered a behavior which is incorrect. When the Action window reaches its minimal width or height and manipulating from left or top, it would end up swimming rather than sticking into position. I will do the changes to address this and update this review with the changes requested. Sorry for the inconvenience.

**Comment by thomas-tu** [comment]
> Content looks good to me. Approving with a request that redundant `private` modifiers be removed prior to merge.
> 
> Edit - Also please change `evt.rawType` -> `evt.type`.

+1 with the requested changes above ^

**Pull request merged**